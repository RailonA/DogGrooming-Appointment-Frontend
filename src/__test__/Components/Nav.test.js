import React from 'react';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom';
import UserProfile from '../../Components/userProfile';
import store from '../../Store/store';

describe('NavBar', () => {
  it('should match the snapshot', () => {
    const wrapper = renderer.create(
      <div className="d-flex justify-content-between p-2 bg-dark text-white">
        <Provider store={store}>
          <BrowserRouter>
            <Link to="/" className="headerText">
              <h2>Pet Groomer</h2>
            </Link>
            <UserProfile />
          </BrowserRouter>
        </Provider>
      </div>,
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
