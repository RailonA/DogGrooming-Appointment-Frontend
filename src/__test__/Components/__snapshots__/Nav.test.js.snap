// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the snapshot 1`] = `
<div
  className="d-flex justify-content-between p-2 bg-dark text-white"
>
  <a
    className="headerText"
    href="/"
    onClick={[Function]}
  >
    <h2>
      Pet Groomer
    </h2>
  </a>
  <div
    className="UserProfile"
  >
    <div>
      <button
        className="mr-2 btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        LOGIN
      </button>
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        SIGN UP
      </button>
    </div>
  </div>
</div>
`;
